packs/smart_items/assets/flying_car_blue_horizontal_platform/data.json

```diff
@@ -21,53 +21,11 @@
           "name": "Go to End",
           "type": "set_state",
           "jsonPayload": "{ \"state\": \"GoingToEnd\" }"
```

Comment:
Shouldn't the actions `Go to Start` and `Go to End` be `allowedInBasicView: true`?